main.js

```diff
@@ -0,0 +1,52 @@
+/*
+* Object for storing global variables
+* possible: input value strings for jquery selections
+* possible: tweet location holder,  geolocation coordiantes when map click
+*
+* */
+
+
+/*
+* Document Ready function
+* holds button click events to activate correct functions
+* */
+
+
+/*
+* Function: getLocation
+* params: *onclick data
+* does: takes location data from google maps api and calls getTweetsByLocation
+* return: nothing;
+* */
+
+
+/*
+*Function: getTweetsbyTag
+* params: takes in text input
+* does: makes api call based on text and stores tweets
+* returns: an array of tweets collected by param search info
+* */
+
+
+/*
+* Function: getTweetsbyLocation,
```

Comment:
Maybe to make this a bit more OOP, have both `getTweetsbyTag` and `getTweetsbyLocation` as two methods of one object that handles all Twitter API calls.